src/components/record/videoEditor/index.js

```diff
@@ -14,6 +14,20 @@ const recordVideoEditorModal = (services, datas, activeTab = false) => {
     let $dialog = null;
     let toolsStream = new Rx.Subject();
 
+    var initialize = function initialize(options) {
+
+        $(document).on('click', '.video-tools-record-action', function (event) {
+            event.preventDefault();
+            var $el = $(event.currentTarget);
+            var idLst = $el.data("idlst");
+            var datas = {}
+            var key = "lst";
+            datas[key] = idLst;
```

Comment:
simplier / easier to follow : 
`var datas = { 'lst': $el.data('idlst') };`
but code is ok to me, **no need to change, keep like this**